tree-sitter.json

```
@@ -0,0 +1,35 @@
+{
+  "$schema": "https://tree-sitter.github.io/tree-sitter/assets/schemas/config.schema.json",
+  "grammars": [
+    {
+      "name": "redscript",
+      "camelcase": "RedScript",
+      "scope": "source.redscript",
+      "file-types": ["redscript"],
+      "injection-regex": "^redscript$",
+      "class-name": "TreeSitterRedscript"
+    }
+  ],
+  "metadata": {
+    "version": "0.1.0",
+    "license": "MIT",
+    "description": "Redscript grammar for tree-sitter",
+    "authors": [
+      {
+        "name": "Jekky"
```

```suggestion
        "name": "jekky"
```
and you can list yotusself